### ISSUE TITLE
expandParent fails when the child node has negative x or y position values

### ISSUE THREAD

**Issue by mlndshh** [wbkd/react-flow#2280]
### Describe the Bug

If you have a node that has a parentNode assigned to it with negative x and/or y values, and have expandParent for the child node, an error is thrown and it results in deletion of the parent node.

Encountered it in an application I'm working on where I assign parent nodes for a node when I drop the child node onto the parent.

### Your Example Website or App

https://codesandbox.io/s/sandpack-project-forked-omrogk

### Steps to Reproduce the Bug or Issue

1. Create a node
2. Create another node with negative x and/or y position values, set its parentNode to the node created earlier along with expandParent

### Expected behavior

The height of the parent should be calculated correctly regardless of the presence of negative position values

### Platform

- OS: macOS 11.6.7
- Browser: Chrome
- Version: 103.0.5060.114

**Comment by mlndshh** [comment]
```
Warning: `NaN` is an invalid value for the `width` css style property.
    at div
    at NodeWrapper (http://localhost:3000/static/js/bundle.js:86998:19)
    at div
    at NodeRenderer (http://localhost:3000/static/js/bundle.js:86109:72)
    at div
    at Viewport (http://localhost:3000/static/js/bundle.js:86805:23)
    at div
    at ZoomPane (http://localhost:3000/static/js/bundle.js:85220:21)
    at FlowRenderer (http://localhost:3000/static/js/bundle.js:85993:23)
    at GraphView (http://localhost:3000/static/js/bundle.js:86829:24)
    at Wrapper (http://localhost:3000/static/js/bundle.js:87222:23)
    at div
    at http://localhost:3000/static/js/bundle.js:87290:20
    at div
    at div
    at Canvas (http://localhost:3000/static/js/bundle.js:443:5)
    at $59d09bcc83651bf9$export$ceb145244332b7a2 (http://localhost:3000/static/js/bundle.js:37673:15)
    at main
    at $b9606c0c41d55371$var$View (http://localhost:3000/static/js/bundle.js:37751:78)
    at Main (http://localhost:3000/static/js/bundle.js:23090:24)
    at div
    at $994c48bfb00b620b$var$Grid (http://localhost:3000/static/js/bundle.js:33816:15)
    at IntlProvider (http://localhost:3000/static/js/bundle.js:88627:43)
    at IntlProvider (http://localhost:3000/static/js/bundle.js:22547:23)
    at Layout (http://localhost:3000/static/js/bundle.js:22886:24)
    at Provider (http://localhost:3000/static/js/bundle.js:99974:7)
    at ReactFlowProvider (http://localhost:3000/static/js/bundle.js:87518:23)
    at div
    at $884c64d19340d345$var$Flex (http://localhost:3000/static/js/bundle.js:33885:15)
    at App (http://localhost:3000/static/js/bundle.js:44:96)
    at $59d09bcc83651bf9$export$ceb145244332b7a2 (http://localhost:3000/static/js/bundle.js:37673:15)
    at div
    at $b9606c0c41d55371$var$View (http://localhost:3000/static/js/bundle.js:37751:78)
    at div
    at ProviderWrapper (http://localhost:3000/static/js/bundle.js:36231:15)
    at $f57aed4a881a3485$export$178405afcd8c5eb (http://localhost:3000/static/js/bundle.js:28332:15)
    at $1051245f87c5981d$export$8214320346cf5104 (http://localhost:3000/static/js/bundle.js:37354:15)
    at $18f2051aff69b9bf$export$a54013f0d02a8f82 (http://localhost:3000/static/js/bundle.js:25270:13)
    at $7167f8da3cce35e4$var$Provider (http://localhost:3000/static/js/bundle.js:36146:21)
    at IntlProvider (http://localhost:3000/static/js/bundle.js:88627:43)
    at IntlProvider (http://localhost:3000/static/js/bundle.js:22547:23)
    at Wrapper (http://localhost:3000/static/js/bundle.js:5018:5)
﻿
```

This is what the stack trace looks like

**Comment by narlark** [comment]
@moklick we are seeing this issue as well. Thoughts?

**Comment by moklick** [comment]
Thanks @mlndshh ! Fixed in v10.3.11

**Comment by mlndshh** [comment]
Thanks!
